otherlibs/dynlink/Makefile

```diff
@@ -38,7 +38,7 @@ endif
 OBJS=dynlinkaux.cmo dynlink.cmo
 
 COMPILEROBJS=\
-  ../../utils/misc.cmo ../../utils/config.cmo \
+  ../../utils/config.cmo ../../utils/misc.cmo \
```

Comment:
This makes me slightly nervous. Maybe `show_config_and_exit` and `show_config_var_and_exit` should be in `config.ml` rather than `misc.ml`?

Reply:
I think that `config` should remain a "pure data" module as much as possible.

I don't think we should add logic to it. Maybe I could create a separate `config_misc` module depending on `config`?

On the other hand, I think that `config` should really be at the bottom of the module dependency graph (especially if we commit to keeping it a pure data module), so I think the change is right in the long term -- although I agree it makes us a bit nervous.

Reply:
I agree that `Config` should be a pure data module and that pure data modules should be at the bottom of the graph, so this is OK after all.